templates/terraform/sweeper_file.go.erb

```diff
@@ -93,7 +115,18 @@ func testSweep<%= sweeper_name -%>(region string) error {
 			nonPrefixCount++
 			continue
 		}
-		deleteTemplate := "<%= object.__product.base_url + object.delete_uri -%>"
+		deleteTemplate := "<%= deleteUrlTemplate -%>"
+		<%  if aggregatedList -%>
+
+		if obj["zone"] == nil {
+			log.Printf("[INFO] %s resource zone was nil", resourceName)
+			return nil
+		}
+		zoneSegs := strings.Split(obj["zone"].(string),"/")
```

Comment:
I think this is just https://github.com/terraform-providers/terraform-provider-google/blob/master/google/self_link_helpers.go#L78, right? Can we use that instead?

Reply:
Didn't see that helper! I actually do the same thing [here](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/sweeper_file.go.erb#L88) to parse the name. I will replace both instances at the same time in a separate PR since it will cause diffs for a ton of existing sweeper files